lib/ueberauth/strategy/amco/oauth.ex

```diff
@@ -10,7 +10,6 @@ defmodule Ueberauth.Strategy.Amco.OAuth do
 
   @defaults [
     strategy: __MODULE__,
-    site: "https://idp.amco.me",
```

Comment:
Are you sure you still don't want a default here in case someone forgets to set something? I think it is good for the "code documenting" purposes. I saw in the other examples and I think in the other part of this, you merge the environment with these defaults, so it would get changed anyway, right?  So something like `provider.example.com` or something similar to show the format and the name you should sue in the config. Plus seeing that in the errors should trigger an aha moment: "Can't connect to upstream provider: provider.example.com"  or whatever the error would be.
Also, do you need to update any other documentation on how to use the module to include this as part of the needed configuration?

Reply:
> Also, do you need to update any other documentation on how to use the module to include this as part of the needed configuration?

Yes, you are right. I updated the README file to show `site` as a required configuration option.

> I saw in the other examples and I think in the other part of this, you merge the environment with these defaults, so it would get changed anyway, right? 

Right, that how it works.

> So something like provider.example.com or something similar to show the format and the name you should sue in the config. Plus seeing that in the errors should trigger an aha moment: "Can't connect to upstream provider: provider.example.com" or whatever the error would be.

IMO having a default that is not valid will be confusing for devs. What I can do is to throw an error if the site config is not present when initializing an oauth client. In the other hand, I want to keep this dependency as short and light as possible, right now the documentation tells us how to configure it and it won't work if the site config is missed (event if the error is not that clear).

What are your thoughts on this?

Reply:
I think throwing an error on a misconfiguration is a good idea. Accomplishes the same goal. I don't think it would be confusing, but the error accomplishes the same thing so I am okay with that route as well. If a dev looks at an error about the app not connecting to example.com and can't figure that out.... 😄

Reply:
I added the error when missing site config.